src/content/development/contribution-guide/_index.en.md

```diff
@@ -58,7 +58,35 @@ To be expanded
 
 #### Pre-Planning
 
-To be expanded
+The week before the release is dedicated to planning the next release.
+Planning accounts for [tasks](#tasks) and [bugs](#bugs) which we intend to solve in the next version.
+Planning meetings will be held, focusing on the [Backlog board].
+
+For a [task](#tasks) to be eligible for the next version, it needs to fulfill these requirements:
+
+* Be part of the [Backlog board].
+* Have a `next-version-candidate` label.
+* Have a description detailing **what** is the issue and optionally **how** is it going to be solved.
+* Have an apropriate sizing label, according to amount of work expected for a single person to completely deliver the task:
+  * *Small*: Work is contained in one sprint and is expected to take less than half the sprint.
+  * *Medium*: Work is contained in one sprint and is expected to take most of the sprint.
+  * *Large*: Work is contained within a release (two-three sprints).
+  * *Extra-Large* = Work can't be contained within a release and would span multiple releases.
+* Any *Large* or *Extra-Large* task must be converted to an [epic](#epics).
+* In case of an [epic](#epics):
+  * Be an issue in the [enhancements] project.
+  * Adhere to the [epic template].
+  * Have a high level break down of the **expected** work, corresponding to the "Definition of Done".
+
+The project team will hold planning meetings, led by the project's "scrum master".
+During these meetings, the project team will:
+
+* Prioritize and assign tasks for the next version.
+  * Only tasks adhering to the described requirements will be considered.
+* Transfer tasks to the [next release board](#current-release-board) according to the capacity of the team to deliver it.
```

Comment:
Shouldn’t we leave tasks on the backlog, and move tasks to the next release board as people start working on them? That’s one of the purposes of a prioritized backlog.

Reply:
In the context of capacity planning, this was discussed in the weekly call; the goal here is to have a per-release backlog separate from the main backlog, because a single backlog hasn’t worked for us so far.

Reply:
I think we need good criteria here but perhaps what we can do is have epics all tracked on the "release board" while other tasks (bugs, small-medium tasks) can be on the general backlog and moved as you say "per demand".

However I don't know if people will be comfortable working with two boards like this (you know we're all lazy lol), so we can try and see.

If you're fine with this idea I'd like to get feedback from the rest of the team as well.

I do think epics must be pre-planned as the EPs will take quite a while to flesh out, and doing them throughout the release (with intention to incorporate into the release itself) hasn't worked well.